scripts/http-jsonp-detection.nse

```diff
@@ -91,13 +92,13 @@ local callback_bruteforce = function(host, port, target)
     path = target
     path = path .. "?" .. p .. "=test"
     response = http.get(host, port, path)
-    if response and response.body and response.status==200 then 
+    if response and response.body and response.status and response.status==200 then 
 
       local status, func
       status, func = checkjson(response.body)
 
-      if status == true then  
-        report = "JSONP endpoint found at " .. path .. ". Callback variable is " .. p
+      if status == true and func == "test" then  
```

Comment:
This check will not identify correctly endpoints with partially controllable function names.